packages/localize-tools/src/tests/analysis.unit.test.ts

```diff
@@ -64,6 +65,7 @@ function checkAnalysis(
 test('irrelevant code', (t) => {
   const src = 'const foo = "foo";';
   checkAnalysis(t, src, []);
+  t.end();
```

Comment:
hm, the requirement for an explicit t.end() is awkward and accident prone compared to just awaiting the return value of the test

wrapper function?